### ISSUE TITLE
keep "Set of" naming consistent

### ISSUE THREAD

**Issue by kodawah** [mtgjson/mtg-sealed-content#118]
we need to avoid separating the "set of" with a dash and to always use number instead of spelling things out

    The Lost Caverns of Ixalan Commander Deck Set of Four 
    Doctor Who Commander Decks Set of Four

    Streets of New Capenna Commander Decks - Set of 5 
    Innistrad Midnight Hunt Commander Decks - Set of 2 
    March of the Machine Commander Decks - Set of 5

are the entries that need a fix

**Comment by kodawah** [comment]
tangential, what do you think about keeping "Box Set" as standard, regardless of number of boxes?

example "The Lord of the Rings Tales of Middle earth Scene Boxes Set of 4" -> "The Lord of the Rings Tales of Middle earth Scene Box Set of 4",

**Comment by axxroytovu** [comment]
I think that's a bigger question. I've been keeping the plural "s" in Set of X products because it reads better:

- Lost Caverns of Ixalan Commander Decks Set of 4
- Ravnica Theme Decks Set of 5
- Strixhaven Prerelease Kits Set of 5

And so specifically those products are called "Scene Box" and there is a set of them, so "Scene Boxes".

Do we remove the plural across the board? I personally prefer having the plural.

**Comment by kodawah** [comment]
I'm not native so some of the nuance may be lost, but I think we decided that the product name should contain the category, and that category is "Scene Box" or "Deck Set" regardless of the amount -- it would simplify the parsing for new names for sure

**Comment by axxroytovu** [comment]
Yeah I think this is a confusion of human readable vs machine readable. I personally think that the product name should be designed to be easily readable by a human, even if that makes the computer parsing more complex. That's why we have all of the categories and subcategories, to allow the computer to parse and sort on categoric inputs that aren't a text string.
